File: Skybuddy_Server/src/modeles/localisation.js
class Localisation {
    constructor (idlocalisation,iduser,idvols,posx,posy,nomuser) {
      this.idlocalisation = idlocalisation;
      this.iduser =iduser;
      this.idvols = idvols;
      this.posx = posx;
      this.posy = posy;
      this.nomuser = nomuser;
    }
    toJson(){
      return {
        'idlocalisation':idlocalisation,
        'iduser':iduser,
        'idvols':idvols,
        'posx':posx,
        'posy':posy,
        'nomuser':nomuser
      };
    }
  }

  module.exports = Localisation;

File: Skybuddy_App/Application/app/src/main/java/com/example/jugid/skybuddy/Objects/Localisation.java
package com.example.jugid.skybuddy.Objects;

import com.google.gson.annotations.SerializedName;

public class Localisation {

    @SerializedName("idlocalisation")
    private String idlocalisation;
    @SerializedName("iduser")
    private String iduser;
    @SerializedName("idvols")
    private String idvols;
    @SerializedName("posx")
    private String posx;
    @SerializedName("posy")
    private String posy;
    @SerializedName("nomuser")
    private String nomuser;

    public String getIdlocalisation() {
        return idlocalisation;
    }

    public String getIduser() {
        return iduser;
    }

    public String getIdvols() {
        return idvols;
    }

    public String getPosx() {
        return posx;
    }

    public String getPosy() {
        return posy;
    }

    public String getNomuser() {
        return nomuser;
    }
}
